nx/lib/nx/defn/graph_splitter/stage.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Nx.Defn.GraphSplitter.Stage do
```

Comment:
```suggestion
defmodule Nx.Defn.Graph.Stage do
```

nitpick: and maybe merge into the file above?